doc/radosgw/dynamicresharding.rst

```diff
@@ -14,17 +14,17 @@ online bucket resharding.
 
 Each bucket index shard can handle its entries efficiently up until
 reaching a certain threshold number of entries. If this threshold is
-exceeded the system can encounter performance issues. The dynamic
+exceeded the system can suffer from performance issues. The dynamic
 resharding feature detects this situation and automatically increases
-the number of shards used by the bucket index, resulting in the
+the number of shards used by the bucket index, resulting in a
```

Comment:
Yeah, I agree with "a" here. This is a really nit-picky quibbly thing that I don't think would present to anyone without a distinguished-level understanding of English, but yeah. I have to agree.

Reply:
There could after all be more than one reduction ;)  It also works with neither.